rllib/core/rl_module/torch/__init__.py

```diff
@@ -1,3 +1,4 @@
 from .torch_rl_module import TorchRLModule
+from .torch_marl_module import TorchMARLModule
 
-__all__ = ["TorchRLModule"]
+__all__ = ["TorchRLModule", "TorchMARLModule"]
```

Comment:
For consistenty: `TorchMultiAgentRLModule`

Reply:
fixed.